linux/ibus-keyman/debian/copyright

```diff
@@ -34,6 +40,25 @@ License: GPL-2+
  License version 2 can be found in the file
  `/usr/share/common-licenses/GPL-2'.
 
+License: GPL-2+
```

Comment:
The License: header here say GPL-2+ but every where else it's GPL-3+, it evidently used to be GPL2+.  Any reason for the change from GPL2+ to GPL3+?  GPL2+ lets it be used with other GPL3+ licensed software but GPL3+ makes it difficult to use with GPL2 licensed stuff.

Reply:
that turned out to be a mistake. That line should be and now is `License: GPL-3+`.
the dbus service `src/keyman-service.c/h` was copied from a GPL-3+ project which is why those files are specified with that license in d/copyright